src/shared/route-urls.ts

```diff
@@ -34,7 +34,7 @@ export enum RouteUrls {
   ViewSecretKey = '/view-secret-key',
 
   // nested routes must have relative paths
-  Activity = 'activity',
+  Activity = '/activity',
```

Comment:
Why this change? Maybe this have unintended consequences?

Reply:
there is ui bug - if refresh page being on activity tab => tab is not active

Reply:
I tested actually, and seems all good

Reply:
I checked and I'm pretty sure I had to change all of these routes [here](https://github.com/leather-wallet/extension/pull/4325/files#diff-a382db38591ec532ccd79d24f26f65e04d7ec0d083b2709d9447a84692b74234) to avoid a console warning but I can't see that now after this change so I think it could be OK.

In future, if you are fixing bugs you should at least have a separate commit message for that fix saying what it is and why in case we need to revisit it